src/Ombi.Core/Authentication/LdapUserManager 2.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
```

Comment:
We have two files `LdapUserManager` and `LdapUserManager 2`?

Reply:
Same bad manipulation. Deleted